src/Presenter/PsAccountsPresenter.php

```diff
@@ -163,6 +171,15 @@ public function present($psxName = 'ps_accounts')
                     'adminAjaxLink' => $this->psAccountsService->getAdminAjaxUrl(),
 
                     'accountsUiUrl' => $this->module->getParameter('ps_accounts.accounts_ui_url'),
+
+                    // 1- open a popup
+                    // 2- callback inject JS close popup & refresh parent
+                    // OU ajax call parent to refresh state (from ajax call to BO state Polling or ??)
+                    'identifyUrl' => $this->oAuth2Service->getOAuth2Client()->getRedirectUri([
+                        'action' => 'identifyPointOfContact',
+                    ]),
```

Comment:
useless here ?

Reply:
I think, we use the identifyUrl here : https://github.com/PrestaShopCorp/ps_accounts/pull/494/files from `getShops` of `ShopProvider`